source/Cosmos.System2/FileSystem/CosmosVFS.cs

```diff
@@ -836,7 +756,11 @@ public override long GetTotalSize(string aDriveId)
         /// <exception cref="Exception">Unable to determine filesystem for path:  + aDriveId</exception>
         public override long GetAvailableFreeSpace(string aDriveId)
```

Comment:
Do we need these wrapper functions? Or it enough if you can get the file system from a drive id to query the data yourself?

Reply:
I think it's better to completely remove the VFSBase and CosmosVFS classes and replace them with a DiskManager class where the user can get all of the disks.